enterprise_access/apps/api/serializers/content_assignments/assignment.py

```diff
@@ -80,6 +79,17 @@ class Meta:
         ]
         read_only_fields = fields
 
+    def get_actions(self, assignment):
+        """
+        Resolves any associated actions to the assignment in chronological order based on created timestamp.
+        """
+        related_actions = assignment.actions.order_by('created').all()
```

Comment:
Tiniest possible nit: it's more conventional to put the `all()` before the `order_by()` (because if you were writing it as raw SQL, the `ORDER BY` would come last).

Reply:
BTW, there's a whole other way to do this, which is to add an `ordering` to the Action model's meta class. https://docs.djangoproject.com/en/4.2/ref/models/options/#ordering

Reply:
Oh, nice. Didn't realize there was a shorthand way to do this via the `Meta` class (TIL)! It's a cleaner solution, will update. Thanks!